src/components/VoteAdder.jsx

```diff
@@ -1,53 +1,48 @@
 import { useEffect, useState } from "react";
-import { updateArticleVotes } from "../api";
+import { fetchArticleById, updateArticleVotes } from "../api";
 
 export default function VoteAdder({ article, setArticle }) {
-  const [hasVoted, setHasVoted] = useState(false);
+  const [isVotedAdded, setIsVoteAdded] = useState(false);
   const [error, setError] = useState(null);
+  const [articleVotes, setArticleVotes] = useState(0);
```

Comment:
I don't think you use this ( and you shouldn't need it, because you can use the article you have stored in state, which it looks like you have done 😄 ). It should be removed